compliance_suite/functions/requestor.py

```diff
@@ -0,0 +1,71 @@
+import json
+import requests
+from compliance_suite.constants.constants import REQUEST_HEADERS
+import polling2
+from compliance_suite.exceptions.ComplianceException import ComplianceException
+
+check_cancel = None
+
+
+def send_request(server, version, endpoint, id_uri_param, query_params, operation, request_body):
+    if id_uri_param:
```

Comment:
I find this a bit too TES-specific. Better to abstract that so that it will replace the correct path param with the correct value.